src/main/java/com/worldwarofants/game/arch/console/ConsoleOutputHandler.java

```diff
@@ -24,17 +31,28 @@ public static void spell(String input) {
                 Thread.currentThread().interrupt();
             }
         }
+        return this;
     }
     /**
      * output the provided string in one line
      */
-    public static void post(String input) {
-        System.out.println(input);   
+    public ConsoleOutputHandler post(String input) {
+        System.out.println(input);
+        return this;
     }
     /**
      * output a line break
      */ 
-    public static void lineBreak() {
-        System.out.println(); 
+    public ConsoleOutputHandler lineBreak() {
+        System.out.println();
+        return this;
+    }
+
+    public void print(){
+        try {
+            finalize();
```

Comment:
This was a slight misunderstanding. :D Calling the method finalize() is in most cases is very bad. You can read about it here. https://docs.oracle.com/javase/10/docs/api/java/lang/Object.html#finalize()

What I meant in general is to do any final steps, if there are any, in the process.

 I dunno what that could be in this particular case. But for example, you could store all of the strings sent through post() in one variable, and on print() you can print it all at once, to sliiightly improve performance, because system.out.println can be considered a slower method than others.

Think about it first though! ;) This is a very cool coding exercise, because whenever spell() is called, it will mess up the order of the messages, if you store all of the posts and print them at once in the end.